includes/definitions.inc.php

```diff
@@ -30,6 +30,22 @@
     $database_db = mysql_select_db($config['db_name'], $database_link);
 }
 
+$config['time']['now']      = time();
+$config['time']['now']     -= ($config['time']['now'] % 300);
```

Comment:
I realise this code existed before but any reason you can see not to put this into one line?